designdocs/provider_boosting_implementation.md

```diff
@@ -255,6 +255,41 @@ pub enum Error<T> {
 }
 ```
 
+### NEW: Events
```

Comment:
In updating the capacity events, noticed these events were added and this seemed like the best place to reference them.